### DISCUSSION TITLE
Two questions relating to backup

### DISCUSSION THREAD

**Discussion by nonsavant** [firefly-iii/firefly-iii#7100] [Firefly III]
Hi all,

Just started backing up my firefly database (docker) but still have the following two questions:

### 1. What is the purpose of the Upload volume - it's empty 

The documentation states to backup the `upload` volume (https://docs.firefly-iii.org/firefly-iii/advanced-installation/backup/). 

However, in my case this volume contains a single .gitignore file.

What is the purpose of this volume and what sort of files should one expect to find there?

### 2. Can I backup my automation rules?

In case of a complete failure and restore, losing all automation rules would be a massive disaster. Is there a way to back them up or export them?

**Comment by JC5** [comment]
https://docs.firefly-iii.org/firefly-iii/advanced-installation/backup/
